src/views/storagemigrations/list/StorageMigrationList.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FC, useMemo } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { modelToRef } from '@kubevirt-utils/models';
+import {
+  DEFAULT_MIGRATION_NAMESPACE,
+  DirectVolumeMigration,
+  DirectVolumeMigrationModel,
+  MigMigration,
+  MigMigrationModel,
+  MigPlan,
+  MigPlanModel,
+} from '@kubevirt-utils/resources/migrations/constants';
+import {
+  getGroupVersionKindForModel,
+  K8sResourceCommon,
+  ListPageBody,
+  ListPageFilter,
+  ListPageHeader,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+import StorageMigrationRow from './components/StorageMigrationRow';
+import useStorageMigrationColumns from './hooks/useStorageMigrationColumns';
+import useStorageMigrationFilters from './hooks/useStorageMigrationFilters';
+import { createMigPlanMap } from './utils';
+
+const StorageMigrationList: FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  const [migplans, loaded, loadError] = useK8sWatchResource<MigPlan[]>({
+    groupVersionKind: getGroupVersionKindForModel(MigPlanModel),
+    isList: true,
+    namespace: DEFAULT_MIGRATION_NAMESPACE,
+    namespaced: true,
+  });
+
+  const [migMigrations] = useK8sWatchResource<MigMigration[]>({
+    groupVersionKind: getGroupVersionKindForModel(MigMigrationModel),
+    isList: true,
+    namespace: DEFAULT_MIGRATION_NAMESPACE,
+    namespaced: true,
+  });
+
+  const [directVolumeMigrations] = useK8sWatchResource<DirectVolumeMigration[]>({
+    groupVersionKind: getGroupVersionKindForModel(DirectVolumeMigrationModel),
+    isList: true,
+    namespace: DEFAULT_MIGRATION_NAMESPACE,
+    namespaced: true,
+  });
```

Comment:
if u want you can extract to a new hook that will combine those calls.. up to you